2025 SC EU seat elections (#1660)
knative/community: elections/2025-SC-EU/README.md

---
title: "Knative Steering Committee End User Seat"
linkTitle: "Steering Committee End User Seat"
weight: 30
type: "docs"
---

# 2025 Knative Steering Committee End User Seat

## Purpose

This document describes the process for filling the end user representative
for Knative Steering Committee. The newly appointed representative will serve a
one (1) year term.

## Background

In 2021 it was decided that after a Steering Committee election there would be an additional election for an end-user seat on the committee. The goal is to give end-users a voice on the Steering Committee.

## Eligibility

In order to be considered for the end user seat, the company the user is
representing must be listed in the [Adopters](../../ADOPTERS.MD) file.

## Schedule

| Date        | Event                                                   |
|-------------|---------------------------------------------------------|

> | January 29  | Announce the intent to appoint end user representative  |

Author: ```suggestion
| February 4  | Announce the intent to appoint end user representative  |
```

---

> title: "Knative Steering Committee end user Seat"
> linkTitle: "Steering Committee end user Seat"

Reviewer: nit casting on
```suggestion
title: "Knative Steering Committee End User Seat"
linkTitle: "Steering Committee End User Seat"
```

weight: 30
type: "docs"
---

> # 2025 Knative Steering Committee end user Seat

Reviewer: ```suggestion
# 2025 Knative Steering Committee End User Seat
```

## Purpose

> This document describes the process for filling the end user Representative

Reviewer: ```suggestion
This document describes the process for filling the end user representative
```

for Knative Steering Committee. The newly appointed representative will serve a
one (1) year term.

## Background

> In 2021 we introduced one end user seat to bring more diversity into the SC
> representation which is to be elected by the SC once the SC election was
> completed.

Reviewer: ```suggestion
In 2021 it was decided that after a Steering Committee election there would be an additional election for an end-user seat on the committee. The goal is to give end-users a voice on the Steering Committee.
```

Rewrote this to make it a bit more clear.

## Eligibility

> In order to be considered for end user seat, the company the user is

Reviewer: ```suggestion
In order to be considered for the end user seat, the company the user is
```

representing must be listed in the [Adopters](../../ADOPTERS.MD) file.

## Schedule

| Date        | Event                                                   |
|-------------|---------------------------------------------------------|
| January 29  | Announce the intent to appoint end user representative  |
| February 14 | All candidate nominations due by 0000 UTC (4pm Pacific) |
| February 19 | Announcement of selection                               |

The one year term would be from February 19, 2025 to until a new representative is chosen in January 2026.

## Candidacy Process

**Nomination**

> If you want to be considered for end user Seat, open a PR against the

Reviewer: ```suggestion
If you want to be considered for end user seat, open a PR against the
```

[knative/community repository](https://github.com/knative/community) to include
your candidate profile in the `/elections/2025-SC-EU` folder, with the following
filename format:

```
candidate-githubid.md
```

This profile should include:

* Your name
* Your company affiliation (employer or otherwise)

> * Your contributions to Knative

Reviewer: ```suggestion
* Your contributions to Knative (if any)
```

* Why you are running

You can find [a sample template in the folder](./nomination-template.md).

### Decision

> The newly elected body will be announced in the Knative Steering Meeting on the
> scheduled announcement date.

Reviewer: ```suggestion
The newly elected representative will be announced in the Knative Steering meeting on the
scheduled date.
```
